src/SFA.DAS.AssessorService.Infrastructure/ApiClients/Roatp/IRoatpApiClientFactory.cs

```diff
@@ -0,0 +1,15 @@
+﻿using SFA.DAS.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AssessorService.Infrastructure.ApiClients.Roatp
+{
+    public interface IRoatpApiClientFactory
```

Comment:
Is there a SFA.DAS.Http interface this can derive from because in the next set of ticket we will need to pass the factory into the base api client which a generic interface if following the factory pattern

Reply:
There is `IHttpClientFactory` with the method  `HttpClient CreateHttpClient();` in SFA.DAS.Http which `IRoatpApiClientFactory` is now inheriting from